shared/dataflow/codeql/dataflow/internal/ContentDataFlowImpl.qll

```diff
@@ -51,6 +51,11 @@ module MakeImplContentDataFlow<LocationSig Location, InputSig<Location> Lang> {
      */
     default predicate isAdditionalFlowStep(Node node1, Node node2) { none() }
 
+    /**
+     * Holds if taint may propagate from `node1` to `node2` in addition to the normal data-flow steps.
+     */
+    default predicate isAdditionalTaintStep(Node node1, Node node2) { none() }
```

Comment:
I don't think we want this; `isAdditionalFlowStep` is already considered a non-value (i.e., taint) step. So I think it suffices to simply remove the line `predicate isAdditionalFlowStep = ContentConfig::isAdditionalFlowStep/2;`.

Reply:
Wasn't the `isAdditionalFlowStep` only considered a taint like step because we encoded this in the flowstate?
Why don't we want both options (one predicate for adding steps that changes the value preserving bit in the flow state and one that doesn't)?
I will make the change that you requested.

Reply:
I don't think we want both, because we also only have `isAdditionalFlowStep` in the normal data flow interface.